tests/unit/test_exec.py

```diff
@@ -8,8 +8,11 @@
 from unittest.mock import patch
 
 from jina import Document, DocumentArray
-from jinahub.segmenter.torch_object_detection_segmenter import TorchObjectDetectionSegmenter
+#from jinahub.segmenter.torch_object_detection_segmenter import TorchObjectDetectionSegmenter
 
+import sys
+sys.path.insert(1, '../..')
```

Comment:
no other executor is doing this

Reply:
I forgot to remove this before pushing, lemme get rid of it